### ISSUE TITLE
FCN训练数据集voc2012只显示366张，正常是1000多张，没有修改代码，训练时候只用了366张。有人遇到这个问题吗？？

### ISSUE THREAD

**Issue by a5336033** [WZMIAOMIAO/deep-learning-for-image-processing#734]
(torch) tian@tian-System:~/deep-learning-for-image-processing/pytorch_segmentation/fcn$ python train.py
Epoch: [0]  [  0/366]  eta: 1:38:35  lr: 0.000000  loss: 4.6789 (4.6789)  time: 16.1613  data: 0.2381  max mem: 4052
Epoch: [0]  [ 10/366]  eta: 0:10:55  lr: 0.000003  loss: 4.8786 (4.7953)  time: 1.8399  data: 0.0218  max mem: 4329

**Comment by WZMIAOMIAO** [comment]
你看的366是step数，不是图片数量。steps = num_imgs / batch_size
